CHANGELOG.md

```diff
@@ -1,12 +1,18 @@
 # Braintree Android SDK Release Notes
 
 ## unreleased
- 
-* PayPal
-  * Fix bug to ensure that `PayPalVaultRequest.userAuthenticationEmail` is not sent as an empty string
+
+* ShopperInsights (BETA)
+    * Add `isPayPalAppInstalled` and `isVenmoAppInstalled` methods
+    * Add `shopperSessionId` parameter to `ShopperInsightsClient`
+* BraintreePayPal
+    * Add `shopperSessionId` to `PayPalCheckoutRequest` and `PayPalVaultRequest`
+    * Fix bug to ensure that `PayPalVaultRequest.userAuthenticationEmail` is not sent as an empty string
+    * Replace `sendPayPalPresentedEvent()` and `sendVenmoPresentedEvent()` with `sendPresentedEvent()`
+    * Replace `sendPayPalSelectedEvent()` and `sendVenmoSelectedEvent()` with `sendSelectedEvent()`
 * ThreeDSecure
-  * Return error if no `dfReferenceId` is returned in the 3D Secure flow
```

Comment:
Nit pick - I think the indentation is correct on the original 3DS entry and the new entries are indented too far